azurelinuxagent/common/version.py

```diff
@@ -196,7 +196,9 @@ def has_logrotate():
 
 AGENT_NAME = "WALinuxAgent"
 AGENT_LONG_NAME = "Azure Linux Agent"
-AGENT_VERSION = '2.2.54.2'
+# Setting the version to 9.9.9.9 to ensure DCR always uses this version and never auto-updates.
```

Comment:
Shouldn't we remove this comment on master? I'm good either way personally just asking more for my understanding.

Reply:
I had the same thought about removing it, but I think we should rather edit it... will do that on develop